flink-libraries/flink-gelly-scala/src/main/scala/org/apache/flink/graph/scala/Graph.scala

```diff
@@ -968,6 +968,46 @@ TypeInformation : ClassTag](jgraph: jg.Graph[K, VV, EV]) {
   }
 
   /**
+   * Performs intersect on the edge sets of the input graphs. Edges are considered equal, if they
+   * have the same source identifier, target identifier and edge value.
+   * <p>
+   * The algorithm computes pairs of matching edges from the input graphs. If the same edge occurs
+   * multiple times in the input graphs, there will be multiple edge pairs to be considered. The
+   * output graph will contain exactly one edge representing all matching pairs.
+   * <p>
+   * Vertices in the output graph will have no vertex values.
+   *
+   * @param graph the graph to perform intersect with
+   * @return a new graph which contains common vertices and edges from the input graphs
```

Comment:
We have "equal", "matching" and "common" to describe the edges of the result. I would stick to one of those to avoid confusion.